htdocs/compta/accounting-files.php

```diff
@@ -715,6 +720,8 @@
 
                 print '<td class="center">'.$data['country_code']."</td>\n";
 
+				print '<td class="center">'.$data['divisa']."</td>\n";
```

Comment:
Note that because this export is for internal accounting, all amount into export is in the same currency, the main currency of company.

Reply:
Hello, yes. That is only additional information for other consultin.